src/components/Book.js

```diff
@@ -1,25 +1,47 @@
-import React from 'react';
+/* eslint-disable camelcase */
+import React, { useEffect } from 'react';
 import { PropTypes } from 'prop-types';
 import { useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/books';
+import { removeBook, DisplayBooks } from '../redux/books/books';
 
 export default function Book({ book }) {
   const dispatch = useDispatch();
 
+  const {
+    title, author, category, item_id,
+  } = book;
+
+  useEffect(() => {
```

Comment:
Though you did a great job with the application, however, `useEffect` hook you are using is causing an infinite loop of dispatching the action. Have a look at [this article](https://stackoverflow.com/questions/53120972/how-to-call-loading-function-with-react-useeffect-only-once). If you have any difficulty figuring it out, kindly leave me a message, but don't forget to tag me.

Reply:
@muneebulrehman Thanks for your review. If I run it once, the page is not refreshed after the remove button click. It would be great if you have time for a zoom call to explain it to me.